src/loggers/console-logger.ts

```diff
@@ -0,0 +1,357 @@
+            const paddedKey = key.padEnd(maxKeyLen, ' ');
+
+            if (raw === null) lines.push(`${indent}${paddedKey}: null`);
+            else if (raw === undefined) lines.push(`${indent}${paddedKey}: undefined`);
+            else if (typeof raw === 'function') lines.push(`${indent}${paddedKey}: <function>`);
+            else if (typeof raw === 'object') {
+              const isArray = Array.isArray(raw);
+              const isEmptyArray = isArray && raw.length === 0;
+              const hasToString =
+                !isArray && typeof raw.toString === 'function' && raw.toString().indexOf('[object') !== 0;
+              const entry = maxIndentReached ? '...' : isEmptyArray ? '[]' : stringify(raw, level + 1, hasToString);
+              lines.push(
+                `${indent}${paddedKey}:${maxIndentReached || hasToString || isEmptyArray ? ' ' : '\n'}${entry}`,
+              );
+            } else lines.push(`${indent}${paddedKey}: ${raw}`);
+
+            skipIndentOnce = false;
+          }
+        }
+      }
+
+      return lines.join('\n');
+    };
+
+    return stringify(message.message);
+  }
+
+  /**
+   * Get a pre-formatted headers list.
+   *
+   * @param message - Log message which may contain an object with headers to be used for formatting.
+   *
+   * @returns String formatted for log entry in console or `undefined` if a log message not related to the network data.
+   */
+  private formattedHeaders(message: NetworkRequestLogMessage | NetworkResponseLogMessage) {
+    if (!message.message.headers) return undefined;
+
+    const headers = message.message.headers;
+    const maxHeaderLength = Object.keys(headers).reduce((max, key) => Math.max(max, key.length), 0);
+
+    return Object.keys(headers)
+      .map((key) => `    - ${key.toLowerCase().padEnd(maxHeaderLength, ' ')}: ${headers[key]}`)
+      .join('\n');
+  }
+
+  /**
+   * Get a pre-formatted body.
+   *
+   * @param message - Log message which may contain an object with `body` (request or response).
+   *
+   * @returns Object with formatted string of form data and / or body for log entry in console or `undefined` if a log
+   * message not related to the network data.
+   */
+  private formattedBody(
+    message: NetworkRequestLogMessage | NetworkResponseLogMessage,
+  ): { body?: string; formData?: string } | undefined {
+    if (!message.message.headers) return undefined;
+
+    let stringifiedFormData: string | undefined;
+    let stringifiedBody: string | undefined;
+    const headers = message.message.headers;
+    const contentType = headers['content-type'] ?? headers['Content-Type'];
+
+    const formData = 'formData' in message.message ? message.message.formData : undefined;
+    const body = message.message.body;
+
+    // The presence of this object means that we are sending `multipart/form-data` (potentially uploading a file).
+    if (formData) {
+      const maxFieldLength = formData.reduce((max, { key }) => Math.max(max, key.length), 0);
```

Comment:
Is it printing all content of file?

considering a potential case that printing multipart form data will make logs a bit unreadable due to it's lengthy content.